bccsp/sw/impl_test.go

```diff
@@ -76,10 +80,10 @@ func TestMain(m *testing.M) {
 		ret := m.Run()
 		if ret != 0 {
 			fmt.Printf("Failed testing at [%d, %s]", config.securityLevel, config.hashFamily)
-			os.Exit(-1)
+			return
 		}
 	}
-	os.Exit(0)
+	code = 0
```

Comment:
this can be removed if `code` var is used

Reply:
i will delete last line 'code = 0'